### PR TITLE
LTA-4-fix-travis

### PR DESCRIPTION
**Pull request by faridfaraji**
This:
----------------
- [x] fix travis.yml

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,33 +3,18 @@ os:
...
-env:
-  - FLUTTER_VERSION=stable
-  - FLUTTER_VERSION=beta
-matrix:
-  allow_failures:
-    - env: FLUTTER_VERSION=beta
+      - fonts-droid-fallback
 before_script:
-  - git clone https://github.com/flutter/flutter.git -b $FLUTTER_VERSION
+  - git clone https://github.com/flutter/flutter.git -b beta
```
Comment (Harkunwar) [inline .travis.yml]: I think we should only use stable?

File: .travis.yml [hunk 2]
```diff
@@ -3,33 +3,18 @@ os:
...
 cache:
   directories:
-    - $HOME/shared/.pub-cache
-notifications:
-  email:
-    brogdon+github@gmail.com
-
-# Only building master means that we don't run two builds for each pull request.
-branches:
-  only: [master]
```
Comment (Harkunwar) [inline .travis.yml:34]: Removing this is causing two builds

**Review by Harkunwar — CHANGES_REQUESTED** [review]
Need changes

**Comment by Harkunwar** [comment]
Can you add a blank test that just returns true?

File: .travis.yml (hunk 1 above)
Comment (faridfaraji) [reply .travis.yml]: most people seem to use beta

**Comment by Harkunwar** [comment]
The current script is just cloning the git and not compiling.

File: .travis.yml [hunk 3]
```diff
@@ -3,32 +3,20 @@ os:
...
   directories:
-    - $HOME/shared/.pub-cache
-notifications:
-  email:
-    brogdon+github@gmail.com
+    - $HOME/.pub-cache
 
+      
 # Only building master means that we don't run two builds for each pull request.
 branches:
```
Comment (faridfaraji) [inline .travis.yml:20]: Is this error cloning happening since i added this?

File: .travis.yml (hunk 1 above)
Comment (Harkunwar) [reply .travis.yml]: let's leave it for now then

File: .travis.yml (hunk 3 above)
Comment (Harkunwar) [reply .travis.yml:20]: But we never reached this error point before so we don't know. I don't think it's this

Comment (Harkunwar) [reply .travis.yml:20]: The error basically is, language not specified

Comment (Harkunwar) [reply .travis.yml:20]: https://docs.travis-ci.com/user/tutorial/#selecting-a-different-programming-language
Change to something from ruby

Comment (Harkunwar) [reply .travis.yml:20]: Try node_js

Comment (faridfaraji) [reply .travis.yml:20]: lets see if this one fails then i change it to node_js, and also I will add that line after to only run one build

Comment (faridfaraji) [reply .travis.yml:20]: Built, its minimal though, which is fine we will add more later also give me a test case to add

**Review by Harkunwar — APPROVED** [review]
LGTM for now. We will need to add more test cases and update this file later. Currently, this is not doing anything

**Pull request merged**